src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/GuardConstants.java

```diff
@@ -0,0 +1,114 @@
+package com.minecolonies.coremod.entity.ai.citizen.guard;
+
+/**
+ * Constants used by the Guard AIs.
+ */
+public final class GuardConstants
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)